### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean [hunk 1]
```diff
@@ -206,11 +206,10 @@ def Basis.parallelepiped (b : Basis ι ℝ E) : PositiveCompacts E where
...
 
--- Porting note: lint complains that this result is a tautology and thus unnecessary
--- @[simp]
--- theorem Basis.coe_parallelepiped (b : Basis ι ℝ E) :
---    (b.parallelepiped : Set E) = parallelepiped b := rfl
--- #align basis.coe_parallelepiped Basis.coe_parallelepiped
+@[simp]
+theorem Basis.coe_parallelepiped (b : Basis ι ℝ E) :
+   (b.parallelepiped : Set E) = _root_.parallelepiped b := rfl
+#align basis.coe_parallelepiped Basis.coe_parallelepiped
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean:212]: Nice catch!

File: Mathlib/AlgebraicGeometry/Morphisms/QuasiSeparated.lean [hunk 2]
```diff
@@ -169,9 +169,8 @@ theorem quasiSeparated_respectsIso : MorphismProperty.RespectsIso @QuasiSeparate
   quasiSeparated_eq_diagonal_is_quasiCompact.symm ▸ quasiCompact_respectsIso.diagonal
 #align algebraic_geometry.quasi_separated_respects_iso AlgebraicGeometry.quasiSeparated_respectsIso
 
-open List in
 theorem QuasiSeparated.affine_openCover_TFAE {X Y : Scheme.{u}} (f : X ⟶ Y) :
-    TFAE
+    List.TFAE
```
Comment (jcommelin) [inline Mathlib/AlgebraicGeometry/Morphisms/QuasiSeparated.lean:173]: Back when this was ported, I complained that I didn't really like the optics of `List.TFAE` and prefer the `TFAE` spelling. That's also what we had in ml3.

I would prefer to have `open List` or maybe `open List (TFAE)` at the top of the file.

**Review by jcommelin — COMMENTED** [review]
bors d+

**Pull request closed without merging**